src/components/AssetItem/AssetItem.js

```diff
@@ -167,6 +168,7 @@ class AssetItem extends PureComponent<Props, StateProps> {
                 <JSwitch
                   onChange={setIsActive}
                   isChecked={isActive}
+                  isDisabled={isActive ? checkETH(address) : false}
```

Comment:
What about `checkETH(address) || isActive` ?

Reply:
I think we need that Ethereum assets will be active all time, or not?

Reply:
we should lock Ethereum to disable when it is already active